### DISCUSSION TITLE
Proposal: Introducing a Stripped-Down Version of Zed?

### DISCUSSION THREAD

**Discussion by psadi** [zed-industries/zed#9487] [Feature Requests]
Hello Team,

I've been reviewing the FAQ section of Zed (https://zed.dev/faq) and found inquiries regarding its pricing and revenue model:

* Will Zed be free?
* How will you make money?

While I appreciate the current rationale behind Zed's monetization strategy, I'm opening this discussion to propose and check the feasibility of the idea of offering two versions of Zed:

**1. Free Version with Limited Features:** This version would provide basic functionalities and cater to users who primarily require a straightforward code/text editor without integrations, such as GitHub, OpenAI, collaboration channels, or Copilot.

**2. Paid Version with Full Features:** The paid version would encompass all the features currently under development for Zed, including existing integrations and future developments that could be monetized. It would be tailored to users who rely on these advanced features for their workflow.

I believe introducing these options would cater to a broader range of users, accommodating both those who need only basic editing capabilities and those who require advanced integrations.

What are your thoughts on this proposal? Do you foresee any potential benefits or drawbacks to implementing a stripped-down version alongside the current development path of Zed?

Looking forward to your perspectives.

Cheers!!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
